oh.ps1

```diff
@@ -523,6 +534,12 @@ function initialize_database {
 }
 
 function start_database {
+	Write-Host "Checking if MySQL server is running..."
+	if ( ( Test-Path "$OH_PATH/$TMP_DIR/mysql.sock" ) -or ( Test-Path "$OH_PATH/$TMP_DIR/mysql.pid" ) ) {
+		Write-Host "MySQL server already running! Exiting."
```

Comment:
There seems to be a move away from saying "MySQL" and using "database" instead which I think is a good idea given MySQL and MariaDB;  so there are other places that should change too?  Agreed?

Reply:
A second pass will be done for the doc